config/routes.rb

```diff
@@ -0,0 +1,86 @@
+Rails.application.routes.draw do
+
+
+  root  'tasks#index'
+  get 'tasks' => 'tasks#index'
+  get '/tasks/new' => 'tasks#new'
+  post  '/tasks'   => 'tasks#create'
+  get '/tasks/update' => 'tasks#update'
```

Comment:
These two routes are incorrect - they do not update a specific task since they don't have the `:id` in the route